fabric/bin/lib/common_ledger.sh

```diff
@@ -112,11 +112,12 @@ ledger_init() {
     try mkdir -p ${FABRIC_STATE_DIR}
     try rm -rf ${FABRIC_STATE_DIR}/*
 
+    try ${FABRIC_BIN_DIR}/cryptogen generate --config=./crypto-config.yaml --output="organizations" 
```

Comment:
i don't think that is necessary since is in the same directory as config files...... And by the way that crypto-config.yaml is not the generated file.

Reply:
> . And by the way that crypto-config.yaml is not the generated file.

In case this was a reference to my comment on this line: i was not referring to crypto-config.yaml but the generated output directory "organizations" ...